Automattic/wp-calypso
# Make Jetpack Search upsell visually consistent with Backup & Scan upsells.

#### Proposed Changes

This PR makes the Jetpack Search upsell component (in Calypso blue) visually similar (with similar fonts, images, styles, etc) as the upsells for both Backup and Scan. Prior to this PR, the Search upsell was not visually consistent with the other upsells (Backup & Scan).

Asana task: 1164141197617539-as-1202234182051627/f

Partially addresses issue: [#59431](https://github.com/Automattic/wp-calypso/issues/59431)

**New Search Upsell:**

![Screenshot on 2022-06-03 at 13-34-47](https://user-images.githubusercontent.com/11078128/171928546-5f5ca917-2738-4242-920c-0d431242026c.png)

**Comparison view:**

**Before**                                 |  **After**
:-------------------------:|:-------------------------:
![](https://user-images.githubusercontent.com/11078128/171929104-582f5fad-902f-4253-bce8-22ebb0234f11.png)  |  ![](https://user-images.githubusercontent.com/11078128/171929196-ab587104-ba94-4689-b1aa-6274c8de052c.png)



#### Testing Instructions

- Spin up this PR in Calypso blue (`yarn start`)
- Go to http://calypso.localhost:3000 and select a self hosted Jetpack site with no purchased plan or products. (or create a Jurassic Ninja site).
- Go to Jetpack --> Search and verify that the new Search upsell appears and that it looks like the "New Search Upsell" image shown above in the description.  You can compare with wordpress.com to see the old vs new.
- Verify the new Search upsell now looks visually consistent with the other upsells at: _Jetpack --> Backup_, and at: _Jetpack --> Scan_.
